ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/main/java/de/adorsys/ledgers/baam/db/domain/BankAccountAccess.java

```diff
@@ -34,7 +34,7 @@ public abstract class BankAccountAccess {
     private AccessCondition conditions; // Conditions on access
 
     @Enumerated(EnumType.STRING)
-    private AccessStatus status; // Active, restricted, or suspended
+    public AccessStatus status; // Active, restricted, or suspended
```

Comment:
Hello @Koufan-De-King  Please make these field **private**... 
Declaring fields as private hides the implementation details and allows you to control how these fields are accessed or modified.
When fields are private, you can provide getter and setter methods to control access, which is the correct way to expose object data.